app/styles/_modules.scss

```diff
@@ -1,3 +1,4 @@
+@import '../bower_components/jquery-modal/jquery.modal.css';
```

Comment:
OK, I have no idea what's going on. It seems like an `@import('foo.css')` should work w/ libsass, per https://github.com/sass/libsass/pull/754 but something funny is happening.

I see the converted `@import` in the /app/styles/main.css:
```css
@import url('/bower_components/jquery-modal/jquery.modal.css');
```

And I see the same thing in .tmp/styles/main.css:
```css
@import url('/bower_components/jquery-modal/jquery.modal.css');
```

But, I don't see that `@import` or "bower_components" or any ".modal" references in the final **dist/styles/8c99447c.en.css** file.